os/android/drmhwctwo.cpp

```diff
@@ -71,6 +73,9 @@ HWC2::Error DrmHwcTwo::Init() {
                                           HWC2::DisplayType::Physical));
 
   displays_.at(HWC_DISPLAY_PRIMARY).Init();
+
+  DrmHwcTwo::GetExplicitSync();
```

Comment:
The "DrmHwcTwo::" is not necessary, GetExplicitSync can be called directly as member function.